2020F-usfca-btec-640.github.io/about.markdown

```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+title: About
+permalink: /about/
+---
```

Comment:
This file should not be a part of the PR. Please remove it so we don't have to deal with merge conflicts.